docs/conf.py

```diff
@@ -68,3 +68,5 @@
         "options": ["source_code_link"],
     },
 }
+# This ensures all needs that are imported show up in the build 'needs.json'
+needs_builder_filter = ""
```

Comment:
this belongs to wherever we have the external need stuff and not into individual conf.py files